Manuals/Altibase_7.3/eng/Altibase SSL TLS User's Guide.md

```diff
@@ -300,20 +306,18 @@ This section discusses how to configure the environment for Altibase SSL.
 -   Step 5: Start the Server
 
 ##### Step 1: Confirm the Installation of OpenSSL and its Library
-It is recommnded to install the OpenSSl toolkit before installing SSL-enabled Altibase. Otherwise, if an Altibase function is used and OpenSSL is not installed, Altibase reports that it is unable to find the OpenSSL library.
-
-Verify that OpenSSL is installed on the server and that it is not infected with the Heartbleed bug. If necessary, install it with the package manager provided by the operating system (e.g., RPM, Red Hat Linux) or download it from http://www.openssl.org/source and compile it manually. 
+It is recommended to install the OpenSSL toolkit before installing SSL-enabled Altibase. Otherwise, if an Altibase function is used and the OpenSSL is not installed, Altibase reports that it is unable to find the OpenSSL library.
```

Comment:
```suggestion
It is recommended to install the OpenSSL toolkit before installing SSL-enabled Altibase. Otherwise, if an Altibase function is used and the OpenSSL is not installed, Altibase will report that it is unable to find the OpenSSL library.
```

Reply:
다른 곳에서 전부 현재형을 써서 이 부분은 현재형을 유지하는 것이 나을 것 같습니다. 순서 상으로 미래의 일이긴 하지만 그것이 중요하지 않고, 알티베이스가 that절을 보고한다, 라는 동작의 설명이니 현재형인 어조가 더 맞을 듯합니다.